index.ts

```diff
@@ -0,0 +1,21 @@
+import { allEvents, FishId, Metadata, Pond, Tag } from '@actyx/pond'
+
+const HelloWorldFish = {
+  fishId: FishId.of('com.example.quickstart', 'quickstart', 0),
+  initialState: 'Hello, World!',
+  onEvent: (_oldState: any, event: any, _metadata: Metadata) => event,
+  where: allEvents,
+}
+
+const main = async () => {
+  const pond = await Pond.default()
+  pond.observe(HelloWorldFish, state => console.log(state))
```

Comment:
`pond.observe(HelloWorldFish, console.log)`?

Reply:
Changed it on purpose since I am worried some people may not know that this works. It's elegant, but possibly confusing to the average developer.